smoke-tests/apps/ConnectionStringOverrides/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/ConnectionStringOverridesTest.java

```diff
@@ -29,9 +29,7 @@
 @UseAgent
 abstract class ConnectionStringOverridesTest {
 
-  @RegisterExtension
-  static final SmokeTestExtension testing =
-      SmokeTestExtension.builder().otelResourceAttributesEnvVar("key1=value1,key2=value2").build();
+  @RegisterExtension static final SmokeTestExtension testing = SmokeTestExtension.create();
```

Comment:
maybe we need an `AksConnectionStringOverridesTest` similar to the `AksRoleNameOverridesTest`?

Reply:
discussed, this is good as-is, I was mistaken